ci/travis.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+set -xe
+
+# Travis likes to fetch submodules recursively, but the build system assumes
+# shalow submodules fetching, so we need to remove all the recursive submodules
+# before proceeding
+git submodule foreach --recursive git submodule deinit --force --all
+
+# Defaults (in case they are not set by the CI)
+F=${F:-production}
+DC=${DC:-dmd1}
+DIST=${DIST:-xenial}
```

Comment:
Also, these defaults could be killed, as this is always expected to be run by travis, we can assume all variables will be defined (and if not it should fail).